src/main/java/frc/robot/robots/subsystem/CANdleSystem.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.robots.subsystem;
+
+import frc.robot.Constants;
+import frc.robot.robots.subsystem.CANdleSystem;
+import edu.wpi.first.wpilibj.util.Color;
+
+public class CANdleSystem
+{
+	private CANdleSystem m_candleSystem;
+
+	public CANdleSystem(int candleid)
+	{
+
+	}
+
+	public CANdleSystem()
```

Comment:
There shouldn't be a different CANdleSystem constructer that uses constants. Remember, subsystems are passed constants